server/routes/api/authenticationProviders.test.ts

```diff
@@ -73,26 +73,25 @@ describe("#authenticationProviders.update", () => {
     expect(res.status).toEqual(400);
   });
 
-  it("should allow admins to disable", async () => {
+  it.only("should allow admins to disable", async () => {
     const team = await buildTeam();
     const user = await buildAdmin({
       teamId: team.id,
     });
-    await team.$create("authenticationProvider", {
+    const googleProvider = await team.$create("authenticationProvider", {
       name: "google",
       providerId: uuidv4(),
     });
-    const authenticationProviders = await team.$get("authenticationProviders");
     const res = await server.post("/api/authenticationProviders.update", {
       body: {
-        id: authenticationProviders[0].id,
+        id: googleProvider.id,
         isEnabled: false,
         token: user.getJwtToken(),
       },
     });
     const body = await res.json();
     expect(res.status).toEqual(200);
-    expect(body.data.name).toBe("slack");
```

Comment:
This test was failing and I think it was wrong.. but I don't know how it was ever passing

Reply:
The `buildTeam` factory creates a slack auth by default, that's how it was passing